api/tests/app/test_camera.py

```diff
@@ -430,7 +431,8 @@ def test_try_get_camera_image_non_existent_id(self, config_rollback):
 
 
 def get_h_inverse(camera_id):
-    path = f"/repo/data/processor/static/data/sources/{camera_id}/homography_matrix/h_inverse.txt"
+    path = f"/repo/api/tests/data/mocked_data/data/processor/static/data/sources/{camera_id}/homography_matrix" \
+           f"/h_inverse.txt"
```

Comment:
I think I changed the default place of this folder to be outside the static folder, the .ini of the tests should change that path (and so the tests), it gets messy having it inside the report's folders